Core/include/Acts/Navigation/DetectorNavigator.hpp

```diff
@@ -415,20 +419,21 @@ class DetectorNavigator {
                 ActsScalar pathToB = b.objectIntersection.pathLength();
                 return pathToA < pathToB;
               });
-    // Set the surface candidate
+    /// Set the surface candidate
```

Comment:
```suggestion
    // Set the surface candidate
```
`///` is only for doxygen